Procfile

```diff
@@ -1 +1 @@
-web: node server.js --verbose
+web: DEBUG=nextstrain:fetch node server.js --verbose
```

Comment:
> A behaviour change is that --verbose no longer includes fetch logging,
> so I adjust the Heroku process invocation to preserve logging of fetches
> in production.  In development, a similar DEBUG value will be necessary
> if fetch logging is still desired.

Should I make a similar change to `package.json` for fetch logging by default for `npm run test:ci`, `… dev`, and `… dev:ssg`?

Reply:
I don't have full context to spot any surprise implications of doing that, but +1 for keeping environments as consistent as possible about things like logging.

Reply:
> Should I make a similar change to package.json for fetch logging by default for npm run test:ci, … dev, and … dev:ssg?

I'd suggest enabling verbose logging¹ for `test:ci` - in part because the logs are sent to a file and we (I) only look at it when things have gone wrong and when things go wrong the more info the better. For `dev` and `dev:ssg` I would skip it. 

¹ I'd personally suggest adding _more_ DEBUG flags - e.g. `DEBUG=nextstrain:*,compression`

Reply:
I enabled `DEBUG=nextstrain:fetch` for `test:ci` as part of [afd9069](https://github.com/nextstrain/nextstrain.org/pull/847/commits/afd90693f655e06da592e27c189b0f7b3febd0d8) and then `DEBUG=nextstrain:*` in a separate commit ([2426cf3](https://github.com/nextstrain/nextstrain.org/pull/847/commits/2426cf3106cf943805faa6966598e215e4db6945)).